source/_integrations/amcrest.markdown

```diff
@@ -20,20 +22,18 @@ There is currently support for the following device types within Home Assistant:
 - Binary Sensor
 - Camera
 - Sensor
+- Events
 
 ## Configuration
 
-To enable your camera in your installation, add the following to your `configuration.yaml` file:
+For configuration go to the `Integrations pane` on your Home Assistant instance.
```

Comment:
- We don't use the term `pane`. Please use `panel` instead.
- No need to mention `on your Home Assistant instance`, that is assumed.
- Provide a click path / more extensive instruction to the user on where to click / what to do.